kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -34,7 +34,7 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
 
   private final val createTime = System.currentTimeMillis()
   private final val handle = OperationHandle(opType, session.protocol)
-  private final val operationTimeout: Long = {
+  protected val operationTimeout: Long = {
```

Comment:
OPERATION_IDLE_TIMEOUT is for operation idle check, it seems to me we don't need to special handle for Laungh engine operation